SofaKernel/modules/SofaBaseMechanics/BarycentricMappers/BarycentricMapperTopologyContainer.inl

```diff
@@ -426,33 +446,33 @@ unsigned int BarycentricMapperTopologyContainer<In,Out,MappingDataType,Element>:
     if(h<0)
         h += m_hashTableSize;
 
-    return h;
+    return static_cast<unsigned int>(h);
```

Comment:
the static cast is needed?

Reply:
I did that to remove the warning about returning an `int` while an `unsigned int` is actually expected. And doing `(unsigned int)h` gives an other warning `old-style cast `.

Reply:
ok 👍